ml-agents/mlagents/trainers/cli_utils.py

```diff
@@ -252,6 +252,15 @@ def _create_parser() -> argparse.ArgumentParser:
         help="Whether to run the Unity executable in no-graphics mode (i.e. without initializing "
         "the graphics driver. Use this only if your agents don't use visual observations.",
     )
+
+    torch_conf = argparser.add_argument_group(title="Torch Configuration")
```

Comment:
`--cpu` option should be moved into this group or just integrate with the device argument

Reply:
Removed `--cpu` since it's obsolete (and added a note in the changelog)